### PR TITLE
Bug OCPBUGS-17513: pmc set boundary hops to 0 and clean up sockets

### PR DESCRIPTION
**Pull request by jzding**

### CODE REVIEW

File: pkg/pmc/pmc.go [hunk 1]
```diff
@@ -24,50 +25,58 @@ var (
...
+	pmcCmd := fmt.Sprintf("pmc -u -b 0 -f /var/run/%s", configFileName)
+	glog.Infof("%s \"%s\"", pmcCmd, cmdStr)
+	e, r, err := expect.Spawn(pmcCmd, -1)
 	if err != nil {
 		return "", []string{}, err
 	}
-	defer e.Close()
+	defer func() {
+		e.SendSignal(syscall.SIGTERM)
+		<-r
```
Comment (aneeshkp) [inline pkg/pmc/pmc.go:36]: need a timeout here , else it will hold the thread forever in worst case

Comment (jzding) [reply pkg/pmc/pmc.go]: added timeout

**Review by aneeshkp — APPROVED** [review]
/lgtm

**Pull request merged**